src/models/ScheduledEvent.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const scheduledEventSchema = new mongoose.Schema(
+  {
+    event: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Event",
+      unique: true,
+    },
+    eventDate: {
+      type: Date,
+    },
+    currentDate: {
+      type: Date,
+      default: new Date.now(),
+    },
```

Comment:
What's the purpose of the `currentDate` field? (I see we have `timestamps: true`, so that should give us a create date, right?)

Reply:
Yeah you are right, I'll change this.